core/modules/user/store/actions.ts

```diff
@@ -315,9 +315,9 @@ const actions: ActionTree<UserState, RootState> = {
     const usersCollection = Vue.prototype.$db.usersCollection
     usersCollection.setItem('current-token', '')
   
-    if (rootStore.state.route.path === '/my-account') {
+    // if (rootStore.state.route.path === '/my-account') {
       // router.push('/')
```

Comment:
Why this got commented? Please revert the original code in here :)

Reply:
Within the Develop Branch the state did not had route defined. So it wont let me build the app. And since it did nothing (the router.push was already commented) I just commented out the whole function until somebody implements it. 

Should I revert it back or leave it there (but it will throw an error "route not defined on rootStore.state")?

Reply:
Ok! It’s fine, there is second redirect in pages/MyAccount.js, asyncData - and this one must stay there :)